spec/factories.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :move do |f|
+    f.game_id "1"
+    f.piece_id "1"

```

Comment:
Don't worry about this stuff yet. We'll add these aspects of the factory once the "Add Game Model" and "Add Piece Model" PRs are merged.